app/models/package_manager/maven.rb

```diff
@@ -17,6 +17,10 @@ def self.package_link(project, version = nil)
       end
     end
 
+    def self.download_url(name, version = nil)
+      "https://repo1.maven.org/maven2/#{name.gsub(/\:|\./, '/')}/#{version}/#{name}-#{version}.jar"
```

Comment:
is there any way to know whether that gsub covers all the cases, like what does the server here actually do... ideally we'd be copying their source code's logic ...

Reply:
would `#{name}` or `#{version}` need any kind of escaping?